runtime/src/accounts_db.rs

```diff
@@ -7586,7 +7586,7 @@ impl AccountsDb {
         let mut accounts_index_root_stats = AccountsIndexRootsStats::default();
         let mut measure = Measure::start("unref_from_storage");
         if let Some(purged_stored_account_slots) = purged_stored_account_slots {
-            let len = purged_stored_account_slots.len();
+            let len = purged_slot_pubkeys.len();
```

Comment:
I'll have to look tomorrow. But at first glance it seems like par_chunks might be an option here. That way we can't accidentally have the wrong number of batches

Reply:
Ah, nevermind. It's a hashset, so we can't use the chunks interface.